lib/models/App.js

```diff
@@ -523,6 +562,28 @@ class App {
 
   }
 
+  executePluginPhase(phase) {
```

Comment:
i think we should default this method to be async, doesn't sound very helpful to keep both lol